src/hydra/hydraClient.test.js

```diff
@@ -73,6 +73,19 @@ describe('map a json-ld document to an admin on rest compatible document', () =>
       expect(reactAdminDocument.comment[0]).toBe('/comments/1');
     });
   });
+
+  describe('transform the JSON-LD document in React Admin document without transform embedded documents to their IRIs', () => {
+    const reactAdminDocument = transformJsonLdDocumentToReactAdminDocument(
+      jsonLdDocument,
+      true,
+      true,
+      true,
+    );
+
+    test('has not transform arrays of embedded documents to their IRIs', () => {
+      expect(reactAdminDocument.comment[0]).not.toBe('/comments/1');
+    });
+  });
 });
```

Comment:
can we have a test where we go through the path where `isIri(key, document[key])` is falsy? Or else, I'm not sure if this function is really useful as used only once.